.github/workflows/testBackendChangesInfraTest.yml

```diff
@@ -9,10 +9,26 @@ on:
     branches: [infra-test]
 
 jobs:
+  extract-api-key:
+    runs-on: ubuntu-latest
+    outputs:
+      api-key: ${{ steps.extract.outputs.api-key }}
+    steps:
+      - name: Extract API key
+        id: extract
+        run: echo "${{ secrets.AHR_API_KEY }}" > api_key.txt
+      - name: Set API key as environment variable
+        shell: bash
+        run: echo "API_KEY=$(cat api_key.txt)" >> $GITHUB_ENV
```

Comment:
maybe this can be simplified (and more secure?) by writing directly from the github secrets to the $GITHUB_ENV ?